deployment/docker-compose/docker-compose.speech.yml

```diff
@@ -10,13 +10,13 @@ services:
       context: ../../optional_components/speech_api
       dockerfile: Dockerfile
       args:
-        PREFERRED_MODEL: ${PREFERRED_MODEL}
+        PREFERRED_MODEL: "small"
```

Comment:
Why was it put here instead of .*.env?

Reply:
That is a bug we discovered, so we need `PREFERRED_MODEL` as a buildtime argument instead of runtime so we can load the models inside the dockerfile image itself, but now that we using different type of .*.env files `docker-compose` cannot recognise it initially during build-time, so @suzinyou and @amiraliemami said to just pass it explicitly right now and we can go through it later